src/librustc_hir/def.rs

```diff
@@ -107,7 +107,7 @@ impl DefKind {
             DefKind::Union => "union",
             DefKind::Trait => "trait",
             DefKind::ForeignTy => "foreign type",
-            DefKind::Method => "method",
+            DefKind::AssocFn => "associated function",
```

Comment:
For the record this is the same length as "associated constant" so it doesn't seem unnecessarily long to me.

Reply:
Plus, it's always correct as opposed to sometimes misleading.